opendbc/car/volkswagen/carcontroller.py

```diff
@@ -79,11 +80,29 @@ def update(self, CC, CS, now_nanos):
 
     if self.CP.openpilotLongitudinalControl:
       if self.frame % self.CCP.ACC_CONTROL_STEP == 0:
-        acc_control = self.CCS.acc_control_value(CS.out.cruiseState.available, CS.out.accFaulted, CC.longActive)
-        accel = float(np.clip(actuators.accel, self.CCP.ACCEL_MIN, self.CCP.ACCEL_MAX) if CC.longActive else 0)
-        stopping = actuators.longControlState == LongCtrlState.stopping
-        starting = actuators.longControlState == LongCtrlState.pid and (CS.esp_hold_confirmation or CS.out.vEgo < self.CP.vEgoStopping)
-        can_sends.extend(self.CCS.create_acc_accel_control(self.packer_pt, self.CAN.pt, CS.acc_type, CC.longActive, accel,
+        # acc is picky about what signals we can send while brake is pressed (e.g. preEnabled at standstill)
+        longActive = CC.longActive and not CS.out.brakePressed
+
+        acc_control = self.CCS.acc_control_value(CS.out.cruiseState.available, CS.out.accFaulted, longActive)
+        accel = float(np.clip(actuators.accel, self.CCP.ACCEL_MIN, self.CCP.ACCEL_MAX) if longActive else 0)
+        stopping = longActive and actuators.longControlState == LongCtrlState.stopping
+        rollout_in_progress = CS.out.vEgo < self.CP.vEgoStopping and accel > 0
```

Comment:
the previous logic would often flip 'starting' to true when slowing to a stop, which is not what the car expects

I'm restricting 'starting' to only positive acceleration here